tensorflow/lite/micro/micro_graph.cc

```diff
@@ -24,6 +24,10 @@ limitations under the License.
 #include "tensorflow/lite/micro/micro_profiler.h"
 #include "tensorflow/lite/schema/schema_generated.h"
 
+#ifndef barrier
+#define barrier() __asm__ __volatile__ ("" ::: "memory")
+#endif
```

Comment:
I'm not sure how to easily check this, but do you know if this will compile on windows? (For our pip packages)
To be sure maybe we can add a `#ifdef LCE_RUN_ON_FPGA` or `#ifdef __GNUC__` or something like that.

Reply:
thanks, good point. making it a NOP on windows.